.github/workflows/e2e.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     runs-on: ubuntu-22.04
     strategy:
       matrix:
-        node-version: [20.x, 22.x]
+        node-version: [22.x]
     steps:
       - name: Checkout
         # https://github.com/actions/checkout/releases/tag/v4.2.2
```

Comment:
In this file (GitHub won't let me comment on the relevant lines) you will want to modify your Cypress run section to look something like this:

```yaml
      - name: Cypress run
        # https://github.com/cypress-io/github-action/releases/tag/v6.7.7
        uses: cypress-io/github-action@f1f0912d392f0d06bdd01fb9ebe3b3299e5806fb
        with:
          start: npm start
          wait-on: 'http://localhost:3000'
          wait-on-timeout: 120
          browser: chrome
          command-prefix: npx happo-e2e -- npx
        env:
          HAPPO_API_KEY: ${{ secrets.HAPPO_API_KEY }}
          HAPPO_API_SECRET: ${{ secrets.HAPPO_API_SECRET }}
```

Reply:
Once we get the happo-e2e command-prefix in there, we should see this message resolved: https://github.com/tuliomir/hathor-wallet/actions/runs/13267916371/job/37039975836?pr=2#step:4:178

Reply:
Thank you, @lencioni ! Everything worked perfectly with your recommendations :+1: